webui/src/pages/repositories/repository/pulls/createPull.jsx

```diff
@@ -67,12 +71,15 @@ const CreatePullForm = ({repo, reference, compare}) => {
             />
         </Form.Group>
         {error && <AlertError error={error} onDismiss={() => setError(null)}/>}
-        <Button variant="success"
-                disabled={!title || !description || loading}
-                onClick={submitForm}>
-            {loading && <><span className="spinner-border spinner-border-sm text-light" role="status"/> {""}</>}
-            Create Pull Request
-        </Button>
+        <div>
+            <Button variant="success"
+                    disabled={!title || !description || loading || isEmptyDiff}
+                    onClick={submitForm}>
+                {loading && <><span className="spinner-border spinner-border-sm text-light" role="status"/> {""}</>}
+                Create Pull Request
+            </Button>
+            {isEmptyDiff && <span className="ms-3 text-warning">Pull requests must include changes.</span>}
```

Comment:
That yellow on white text is really hurting my eyes:
![image](https://github.com/user-attachments/assets/7b086e5f-ccee-4dc0-9a34-ee87705b8144)
Let's use an Alert element like we do in all other locations or some other format that fits the UI